css/unified-search.scss

```diff
@@ -0,0 +1,23 @@
+.unified-search {
+	.conversation-icon {
+		background-color: var(--color-background-darker);
```

Comment:
Right, haven't thought of this :thinking: 
Ideally you shouldn't have to apply custom styling to a properly designed api.
(nonetheless what you did is very clean!)

@georgehrke @juliushaertl thoughts?

Reply:
Well this is how the talk icons work. we use "32px round background + 22px icon on top" as our "icon"